### DISCUSSION TITLE
欢迎来到openmldb讨论组！

### DISCUSSION THREAD

**Discussion by imotai** [4paradigm/OpenMLDB#5] [General]
## 👋 欢迎!
  我们将用github discussion作为OpenMLDB社区的基础. 在这里你可以:
  * 交流你关注的问题
  * 分享对fedb的一些想法
  * 与社区其他成员交流互动
  * 与大家共建一个公平交流的技术社区 💪！

  为了融入社区，大家可以在这个讨论下面做一个简单自我介绍！

**Comment by auula** [comment]
> ## 👋 欢迎!
> 我们将用github discussion作为OpenMLDB社区的基础. 在这里你可以:
> 
> * 交流你关注的问题
> * 分享对fedb的一些想法
> * 与社区其他成员交流互动
> * 与大家共建一个公平交流的技术社区 💪！
> 
> 为了融入社区，大家可以在这个讨论下面做一个简单自我介绍！

community discussion looks good.
